### PR TITLE
test: #188 confirm

### PR DESCRIPTION
**Pull request by Alex89198900**
1. Add test to confirm

### CODE REVIEW

File: mw-webapp/src/component/confirm/Confirm.cy.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        }
+        onOk={() => {}}
+        okText="Test confirm"
+      />
+      ,
+    );
+  });
+
+  it("should not exist by default", () => {
+    cy.get(getDataCy(CONFIRM_CY.dataCyContent.dataCyContent))
```
Comment (tritonJS826) [inline mw-webapp/src/component/confirm/Confirm.cy.tsx:42]: lets use helper "getDataCy" for getting elements by cy attribute

File: mw-webapp/src/component/confirm/Confirm.cy.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      .should("not.exist");
+  });
+
+  it("should render content", () => {
+    cy.get(getDataCy(CONFIRM_CY.dataCyTrigger))
+      .click();
+    cy.get(getDataCy(CONFIRM_CY.dataCyContent.dataCyContent))
+      .should("exist");
+    cy.get(getDataCy(CONFIRM_CY.dataCyContent.dataCyClose))
```
Comment (tritonJS826) [inline mw-webapp/src/component/confirm/Confirm.cy.tsx:82]: unneccessary linewe should not close modal in this test

File: mw-webapp/src/component/confirm/Confirm.cy.tsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        }
+        content={
+          <div>
+            Confirm content
+          </div>
+        }
+        onOk={() => {}}
+        okText="Test confirm"
+      />
+      ,
```
Comment (tritonJS826) [inline mw-webapp/src/component/confirm/Confirm.cy.tsx:37]: unneccessary line

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/component/confirm/Confirm.cy.tsx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+    cy.get(getDataCy(CONFIRM_CY.dataCyTrigger))
+      .click();
+    cy.get(getDataCy(CONFIRM_CY.dataCyContent.dataCyContent))
+      .should("exist");
+    cy.get(getDataCy(CONFIRM_CY.btnCancel))
+      .click();
+    cy.get(getDataCy(CONFIRM_CY.dataCyContent.dataCyContent))
+      .should("not.exist");
+  });
+  it("should open on trigger click and close by clicking on ok button", () => {
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/confirm/Confirm.cy.tsx:66]: Maybe we should check  do onClick works by clicking onOk?
And is it possible to check do OnEnter click work?

File: mw-webapp/src/component/confirm/Confirm.tsx [hunk 5]
```diff
@@ -2,10 +2,26 @@ import {ReactElement, useRef, useState} from "react";
...
+  /**
+   * Data attribute for cypress testing
+   */
+  btnCancel?: string;
+
+  /**
+   * Data attribute for cypress testing
+   */
+  btnOk?: string;
+}
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/confirm/Confirm.tsx:23]: never use short name please) better use onOk, onCancel

File: mw-webapp/src/component/confirm/Confirm.tsx [hunk 6]
```diff
@@ -2,10 +2,26 @@ import {ReactElement, useRef, useState} from "react";
...
 import {HorizontalContainer} from "src/component/horizontalContainer/HorizontalContainer";
-import {Modal} from "src/component/modal/Modal";
+import {Cy, Modal} from "src/component/modal/Modal";
 import {KeySymbols} from "src/utils/KeySymbols";
 import styles from "src/component/confirm/Confirm.module.scss";
 
+/**
+ * Data attributes for cypress testing
+ */
+interface cyDataConfirm extends Cy {
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/confirm/Confirm.tsx:12]: Start name of interfaces with big letter

Comment (Alex89198900) [reply mw-webapp/src/component/confirm/Confirm.tsx:12]: Done

File: mw-webapp/src/component/confirm/Confirm.tsx (hunk 5 above)
Comment (Alex89198900) [reply mw-webapp/src/component/confirm/Confirm.tsx:23]: Done

File: mw-webapp/src/component/confirm/Confirm.cy.tsx (hunk 3 above)
Comment (Alex89198900) [reply mw-webapp/src/component/confirm/Confirm.cy.tsx:37]: Done

File: mw-webapp/src/component/confirm/Confirm.cy.tsx (hunk 4 above)
Comment (Alex89198900) [reply mw-webapp/src/component/confirm/Confirm.cy.tsx:66]: Done

File: mw-webapp/src/component/confirm/Confirm.cy.tsx (hunk 2 above)
Comment (Alex89198900) [reply mw-webapp/src/component/confirm/Confirm.cy.tsx:82]: Done

File: mw-webapp/src/component/confirm/Confirm.cy.tsx (hunk 1 above)
Comment (Alex89198900) [reply mw-webapp/src/component/confirm/Confirm.cy.tsx:42]: Didn't understand. I guess that's what I do

**Review by Ekaterina1994 — APPROVED** [review]

**Pull request merged**